R/03-annotate.R

```diff
@@ -122,6 +122,11 @@ gimap_annotate <- function(.data = NULL,
       "double_targeting"
     )))
 
+  if (gimap_dataset$filtered_data$filter_step_run){
```

Comment:
Same comment I have on the previous PR but might be easier to revise after merge